doc/api/stream.md

```diff
@@ -1635,10 +1635,13 @@ run().catch(console.error);
 ```
 
 The `pipeline` API also supports async generators:
+<!-- YAML
+added: v13.10.0
```

Comment:
This should be added ins the YAML comment starting on line 1573 under `changes`.

Reply:
@kasvtv Could you update the PR with this?

Reply:
there's already a changes: section about it